Resources/Prototypes/Entities/Mobs/NPCs/prawn.yml

```diff
@@ -0,0 +1,270 @@
+- type: entity
+  name: Prawn
+  id: MobPrawn
+  parent: SimpleMobBase
+  description: A miserable pile of seafood.
+  components:
+  - type: EnsnareableVisualizer
+  - type: Climbing
+  - type: Cuffable
+  - type: CombatMode
+    canDisarm: true
+  - type: SlowOnDamage
+    speedModifierThresholds:
+      60: 0.7
+      80: 0.5
+  - type: HeatResistance
+  - type: DamageVisuals
+    thresholds: [ 20, 40, 100 ]
+    targetLayers:
+      - "enum.HumanoidVisualLayers.Chest"
+      - "enum.HumanoidVisualLayers.Head"
+      - "enum.HumanoidVisualLayers.LArm"
+      - "enum.HumanoidVisualLayers.LLeg"
+      - "enum.HumanoidVisualLayers.RArm"
+      - "enum.HumanoidVisualLayers.RLeg"
+    damageOverlayGroups:
+      Brute:
+        sprite: Mobs/Effects/brute_damage.rsi
+        color: "#FF0000"
+      Burn:
+        sprite: Mobs/Effects/burn_damage.rsi
+  - type: InteractionPopup
+    successChance: 1
+    interactSuccessString: hugging-success-generic
+    interactSuccessSound: /Audio/Effects/thudswoosh.ogg
+    messagePerceivedByOthers: hugging-success-generic-others
+  - type: MovementSpeedModifier
+    baseWalkSpeed : 3
+    baseSprintSpeed : 3.7
+  - type: InputMover
+  - type: MobMover
+  - type: Reactive
+    groups:
+      Flammable: [ Touch ]
+      Extinguish: [ Touch ]
+      Acidic: [Touch, Ingestion]
+    reactions:
+    - reagents: [Water, SpaceCleaner]
+      methods: [Touch]
+      effects:
+      - !type:WashCreamPieReaction
+  - type: Icon
+    sprite: Mobs/Species/Prawn/parts.rsi
+    state: full
+  # Other
+  - type: RandomMetadata
+    nameSet: NamesPrawn
+  - type: StandingState
+  - type: Sprite
```

Comment:
can't comment on textures so i'll put this here. You have both individual parts textures as well as whole body sprites. Since you're assembling this from the individual body parts, you do not need to have those additional textures. Look at how the other species do this.